### DISCUSSION TITLE
SolarEdge not visible in Devices

### DISCUSSION THREAD

**Discussion by rpycke** [addiejanssen/domoticz-solaredge-modbustcp-plugin#64] [Q&A]
I installed the SolarEdge inverter (type 3 installation: using Modbus TCP) following the installation description in https://www.domoticz.com/wiki/SolarEdge .
At the end of this installation procedure I get the message ‘Hardware created, devices can be found in the devices tab!’
But I don’t see a SolarEdge device in the Devices tab.
What can be wrong?

**Comment by smmadge** [comment]
You haven't provided any debuglogs so no one can see what might be wrong. What you can do and what helped me is to follow the following solution of a problem you might have:
https://github.com/addiejanssen/domoticz-solaredge-modbustcp-plugin/discussions/62
